app/views/comments/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for ([current_user, @post, @post.comments.build]) do |form| %>
+<div class="row">
+  <div class="col-md-3"></div>
+  <div class="col-md-6">
+    <div class="form-group">
+      <div class="input-group">
+        <%= form.label :message %><br>
+        <%= form.text_field :message %>
+      </div>
+    </div>
+
+
+
+    <div class="btn">
+      <%= form.submit %>
+    </div>
+    <% end %>
```

Comment:
It's useful to find the command in your editor that automatically correctly indents a whole file.  It avoids indentation mistakes, and saves time.